consensus/istanbul/core/roundstate_db.go

```diff
@@ -40,6 +41,7 @@ const (
 	dbVersionKey = "version"  // Version of the database to flush if changes
 	lastViewKey  = "lastView" // Last View that we know of
 	rsKey        = "rs"       // Database Key Pefix for RoundState
+	rcvdKey      = "rcvd"     // Database Key Prefix for rcvd messages from the RoundState (split saving)
```

Comment:
why not have differente keys for prepares, commits and parent commits?

Reply:
I believe they shouldn't have an outstanding size, though I agree that further splitting is better. I'll check how the code looks with it.